tests/test_models/test_simple_models.py

```diff
@@ -178,7 +188,7 @@ def test_deadline_model_predict_fail_not_enough_context(simple_df):
     model = DeadlineMovingAverageModel(window=1000)
     model.fit(simple_df)
     with pytest.raises(ValueError, match="Given context isn't big enough"):
-        _ = model.forecast(simple_df, prediction_size=7)
+        _ = model.predict(simple_df, prediction_size=7)
```

Comment:
Why did you replace method here?

Reply:
Look at the test name, it was an artifact of copypasta.